deploy/kubernetes/overlays/examples/filter-metrics.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1beta2 # for versions before 1.8.0 use extensions/v1beta1 [apps/v1beta2]
+kind: DaemonSet
+metadata:
+  name: cadvisor
+spec:
+  template:
+    spec:
+      containers:
+      - name: cadvisor
+        command:
+          - /usr/bin/cadvisor
```

Comment:
nit: prefer
```yaml
command:
  - /usr/bin/cadvisor
  - --logtostderr
arguments:
- --disable_metrics=percpu,disk,network,tcp,udp
- --docker_only
```

Alternatively, move the command to the base template

Reply:
removed `command` entirely, as those are the default args specified by the Dockerfile for cadvisor